map_analysis_2d/ui/main_window.py

```
@@ -1539,9 +1549,48 @@ def _cache_peak_fitting_config_from_panel(self):
+            serializable = dict(config)
+            region = serializable.get('region')
+            if isinstance(region, tuple):
+                serializable['region'] = list(region)
+            bounds = serializable.get('bounds')
+            if isinstance(bounds, tuple) and len(bounds) == 2:
+                serializable['bounds'] = [list(bounds[0]), list(bounds[1])]
+            from core.config_manager import ConfigManager
+            cfg = ConfigManager()
+            cfg.set('map_analysis.last_peak_fitting_config', serializable)
+            self._saved_peak_fitting_config = config
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)